wp-content/themes/pub/wporg-learn-2020/template-parts/component-author.php

```diff
@@ -1,9 +1,27 @@
+<?php
+/**
+ * Template part for display author information
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package WPBBP
+ */
+
+$display_name = get_the_author_meta( 'display_name' );
+$nickname = get_the_author_meta( 'nickname' );
```

Comment:
I think you want `user_nicename` here instead of `nickname`. The nickname isn't necessarily the same as the "user name" that you see on their wporg profile.

Reply:
Here I'm using the nickname as a hack to handle the @{username} to match the designs. I'm not sure the display name here will match the .org profile name?

Reply:
Yeah, `display_name` and `nickname` are both fields where the user can enter something arbitrary. To match the `@{username}` in the design, you'll want to use `user_nicename` or `user_login`.